src/Entity/UserManager.php

```diff
@@ -21,4 +23,31 @@ public function isPasswordValid(UserInterface $user, string $password, PasswordE
   {
     return $encoder->isPasswordValid($user->getPassword(), $password, $user->getSalt());
   }
+
+  public function getMappedUserData(array $raw_user_data): array
+  {
+    $response_data = [];
+
+    foreach ($raw_user_data as $user)
+    {
+      try
+      {
+        $country = Countries::getName(strtoupper($user->getCountry()));
+      }
+      catch (MissingResourceException $e)
+      {
+        $country = '';
+      }
+      array_push($response_data, [
+        'username' => $user->getUsername(),
+        'id' => $user->getId(),
+        'avatar' => $user->getAvatar(),
+        'projects' => count($user->getPrograms()),
```

Comment:
Maybe we should call it `project_count`?
```suggestion
        'project_count' => count($user->getPrograms()),
```